.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Global owners
+* @embrace-io/team-react-native
```

Comment:
It says this team is not public or doesn't have write access? Is it because you haven't created it yet?